csrc/ops/schedule.h

```diff
@@ -0,0 +1,29 @@
+// clang-format off
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2025-present NVIDIA CORPORATION & AFFILIATES.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+// clang-format on
+#pragma once
+
+#include <exceptions.h>
+#include <visibility.h>
+
+#include <ir/interface_nodes.h>
+#include <ops/utils.h>
+#include <type.h>
+
+namespace nvfuser {
+
+// The launch dependent grid instruction allows the driver to start the
+// dependent grid in programmatic dependent launch. This is commonly used after
+// computation is finished but before storing results to global memory.
+NVF_API TensorView* launch_dependent_grid(std::vector<Val*> inputs);
```

Comment:
Why is this necessary to be exported as a user-facing op?

Reply:
I understand this may make sense as an initial incremental step, so I am not opposed to it. However, since it deviates from the main design principle, it warrants extra attention. At a minimum, please add a comment explaining the rationale and the context you mentioned above.